### PR TITLE
Generic credentials

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

**Comment by fredericosilva** [comment]
@ad-m rebase request! :)

**Comment by ad-m** [comment]
@fredericosilva , oh no....done!

**Comment by ad-m** [comment]
> Resource case is not unified
I noticed it earlier ( https://app.asana.com/0/1122798235637646/1128299105304288 ), but fixed by self now.

File: bin/generic/credential/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = (resource) => {
         name: 'credential',
         defaultQuery: '[].{id:_id,name:name,type:type}',
         url: args => `${resource.url(args)}/${args[resource.name]}`,
-        commands: ['list', 'show', 'delete'],
+        commands: [],
         options: options,
         plugins: resource.plugins,
         description: `Manage your credentials to ${resource.title}`,
```
Comment (fredericosilva) [inline bin/generic/credential/index.js]: change 'to' to 'of'

**Pull request merged**